app/components/pages/AuthorGuide/TheEditorialProcess.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { H1 } from '@pubsweet/ui'
+
+import Paragraph from '../../ui/atoms/Paragraph'
+
+const AuthorGuide = props => (
+  <div>
+    <H1>The Editorial Process</H1>
+    <Paragraph>
+      eLife publishes promising research across the life sciences and
+      biomedicine. Before you submit your work, please note that eLife is a very
+      selective journal that aims to publish work of the highest scientific
+      standards and importance. Leading academic researchers evaluate new
+      submissions and approximately two-thirds are returned to the authors
+      without further peer review. Approximately half of the articles that are
+      selected for peer review go on to be published.
+    </Paragraph>
+  </div>
+)
+
+export default AuthorGuide
```

Comment:
Did you mean to include this in this PR?

Reply:
Oops! Removed :+1: